test/vmtester/tests.cpp

```diff
@@ -9,6 +9,12 @@
 #include <array>
 #include <cstring>
 
+// Declarations of functions from example host:
+
+evmc_context* example_host_create_context();
+void example_host_destroy_context(evmc_context* context);
```

Comment:
Why not create a header for it now?

Reply:
I don't want it to be a part of en example.